packages/node/src/cli.js

```diff
@@ -285,6 +285,13 @@ const main = async (args = hideBin(process.argv)) => {
         global: true,
         group: BEHAVIOR_GROUP,
       },
+      'scuttle-global-this': {
```

Comment:
I'd prefer just `scuttle`, but feel free to use an alias if you disagree 😄

Reply:
fixed aa97a196e